### ISSUE TITLE
Visited Link color (Purple 500, #8D6E9F) has slightly low contrast on white

### ISSUE THREAD

**Issue by stevennguyen-okta** [okta/odyssey#559]
## Detailed description

> Is this issue related to a specific component?
Nope, just the color

> What did you expect to happen? What happened instead?
I expected the visited link to have sufficient contrast on white. Instead, it feels slightly too light. https://design-docs.trexcloud.com/components/link.html 

> What browser & version are you experiencing this in?

> Is there a specific WCAG checkpoint or ARIA guideline being violated?
WCAG AA and AAA for normal text. It seems to have a ratio of 4.31:1 per https://webaim.org/resources/contrastchecker/?fcolor=8D6E9F&bcolor=FFFFFF

> Is this issue related to a specific assistive technology?
Nope

## Screenshots
![image](https://user-images.githubusercontent.com/39841526/85887306-5d850000-b79c-11ea-9e69-4e7c6e5d366f.png)
![image](https://user-images.githubusercontent.com/39841526/85887323-64ac0e00-b79c-11ea-8143-790d4d0554e0.png)

If relevant, please include screenshots to help illuminate the issue.
